Facebook-DP-Downloader/fb_dp_downloader.py

```diff
@@ -0,0 +1,32 @@
+"""
+    Facebook-DP-Downloader
+        Download the profile picture of any public profile on facebook
+        by just having it's fbid
+    
+"""
+
+import os
+import requests
+
+url="https://graph.facebook.com/{}/picture?type=large"
+""" This url is the url provided by the facebook graph api
+which helps to get to the profile picture of the corresponding fbid
+{}==fbid
+fbid denotes the unique user id of the facebook profile
+whose profile we are requesting
+"""
+
+path = os.getcwd()
+# get the path of the current working directory
+
+if not "fb_dps" in os.listdir(path):
+    os.mkdir("fb_dps")
+"""checks if the folder exists in the current working directory.
+If it does not exist, then it gets created
+"""
+
+for i in range(4,11):
+    # range here can vary, but the number should be a valid fbid
+    result=requests.get(url.format(i))
+    with open("fb_dps/{}_img.jpg".format(i),"wb") as file:
+        file.write(result.content)
```

Comment:
What is this logic? With this code, only images specified by you will be downloaded. Take user id input from the user and then download.

Also, handle this exception:

`{"error":{"message":"An access token is required to request this resource.","type":"OAuthException","code":104,"fbtrace_id":"ApCkvkg1LmRyLUbzMdgZxiq"}}`

Reply:
@kaustubhgupta 

![updated](https://user-images.githubusercontent.com/72334601/118871831-9292c780-b905-11eb-9ec3-63dcbfd6a550.PNG)

will this be fine for the last change requested..?

Reply:
If the program is not giving extra errors and working at your end then it's fine. Also, why are you doing this in a loop? The user will pass the Facebook id and only that picture should be downloaded. No bulk downloading!

Reply:
> If the program is not giving extra errors and working at your end then it's fine. Also, why are you doing this in a loop? The user will pass the Facebook id and only that picture should be downloaded. No bulk downloading!

okay 👍